src/pages/[domain]/admin/configure-place.tsx

```diff
@@ -1,15 +1,246 @@
+          subdomain: values.subdomain,
+          logo: values.logo,
+          name: builderPlaceData.name,
+          ownerTalentLayerId: builderPlaceData.ownerTalentLayerId,
+          palette,
+          owners: builderPlaceData.owners,
+          status: 'validated',
+          signature,
+        });
+        router.push(`${window.location.protocol}//${values.subdomain}/dashboard`);
+      } catch (e: any) {
+        console.error(e);
+      } finally {
+        setSubmitting(false);
+      }
+    }
+  };
+
   return (
-    <div>
-      <h1>Configure your place</h1>
-      <Link href='/admin/custom-domain'>Setup your own domain</Link>
-    </div>
+    <>
+      <HirerProfileLayout step={0}>
+        <Formik
+          initialValues={initialValues}
+          enableReinitialize={true}
+          onSubmit={handleSubmit}
+          validationSchema={validationSchema}>
+          {({ isSubmitting, setFieldValue, values }) => (
+            <Form>
+              <div className='grid grid-cols-1 gap-6'>
+                <label className='block'>
+                  <span className='text-stone-800 font-bold text-md'>custom domain</span>
+                  <Field
+                    type='text'
+                    id='subdomain'
+                    name='subdomain'
+                    className='mt-1 mb-1 block w-full rounded-xl border-2 border-gray-200 bg-midnight shadow-sm focus:ring-opacity-50'
+                    placeholder={`.${process.env.NEXT_PUBLIC_ROOT_DOMAIN}`}
+                  />
+                </label>
+                <span className='text-red-500'>
+                  <ErrorMessage name='subdomain' />
+                </span>
+                <label className='block'>
+                  <span className='text-stone-800 font-bold text-md'>logo</span>
+                  <input
+                    type='file'
+                    id='logo'
+                    name='logo'
+                    onChange={async (event: any) => {
+                      await uploadImage(
+                        event.currentTarget.files[0],
+                        setFieldValue,
+                        setLogoErrorMessage,
+                        'logo',
+                        setLogoLoader,
+                        user?.handle,
+                      );
+                    }}
+                    className='mt-1 mb-1 block w-full rounded-xl border-2 border-gray-200 shadow-sm focus:ring-opacity-50'
+                    placeholder=''
+                  />
+                  {logoLoader && <Loading />}
+                  {!!values.logo && (
+                    <div className='flex items-center justify-center py-3'>
+                      <img width='300' height='300' src={values.logo} alt='' />
+                    </div>
+                  )}
+                </label>
+                <span className='text-red-500'>
+                  <p>{logoErrorMessage}</p>
+                </span>
+
+                {palette && (
```

Comment:
we style need to allow to use a default one

Reply:
Done